### PR TITLE
Create Alert Events from Internal IP Address.txt

### PR DESCRIPTION
**Pull request by jing-microsoft**
Determines MachineId from internal IP address and outputs all alerts in events table associated to the MachineId

### CODE REVIEW

File: General queries/Alert Events from Internal IP Address.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+let TimeRangeEnd = PivotTime+15m; // 15 Minutes After Pivot Time
+let IPAddress = "10.0.0.34";  // internal IP address to search
+// Locate MachineIDs associated with IP
+let FindMachineIDbyIP = MachineNetworkInfo
+| where EventTime between ((TimeRangeStart) ..TimeRangeEnd) 
+	and IPAddresses contains strcat("\"", IPAddress, "\"") 
+	and NetworkAdapterStatus == "Up"
+| project ComputerName, MachineId, EventTime, IPAddresses;
+// Query Alerts matching MachineIDs
+AlertEvents
```
[the remark this answers is not in the document]
Comment (jing-microsoft) [reply General queries/Alert Events from Internal IP Address.txt]: Nice. Can we add it as optional with a comment double slash.

[the remark this answers is not in the document]
Comment (jing-microsoft) [reply General queries/Alert Events from Internal IP Address.txt]: Added This

// Summarizes alerts by AlertId with min and max event times
| summarize Title=any(Title), min(EventTime), max(EventTime), TotalAlerts=count(AlertId) by AlertId

File: General queries/Alert Events from Internal IP Address.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Query # 1 - Determines MachineId from internal IP address and outputs all alerts in events table associated to the MachineId
```
[the remark this answers is not in the document]
Comment (jing-microsoft) [reply General queries/Alert Events from Internal IP Address.txt]: Removed

File: General queries/Alert Events from Internal IP Address.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+// Example use case is Firewall determines Internal IP with suspicious network activity. Query WDATP based on date/time and Internal IP and see associated alerts for the endpoint.
```
[the remark this answers is not in the document]
Comment (jing-microsoft) [reply General queries/Alert Events from Internal IP Address.txt]: I will add comment a link once its published to the Machine info from IP Address.txt

File: General queries/Alert Events from Internal IP Address.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (TomerAlpert) [reply General queries/Alert Events from Internal IP Address.txt]: Great - but please remove TotalAlerts (as you count by AlertId - so it should be 1...)
Also, would be nice to include ComputerName (could be after the by, or before the by with an any())

**Pull request merged**